heron/stmgr/src/cpp/manager/stmgr-client.cpp

```diff
@@ -192,6 +194,12 @@ bool StMgrClient::SendTupleStreamMessage(proto::stmgr::TupleStreamMessage& _msg)
   if (IsConnected()) {
     SendMessage(_msg);
     return true;
+  } else if (droptuples_upon_backpressure_ && HasCausedBackPressure()) {
+    if (++ndropped_messages_ % 100 == 0) {
+      LOG(INFO) << "Dropping " << ndropped_messages_ << "th tuple message to stmgr "
```

Comment:
The drop count is for both not connected and backpressure, so the log could be misleading. Just a concern to me, not a real issue. Your call.

Reply:
I think as long as we are counting drops, we should be ok